sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -286,6 +287,10 @@ def _read_side_inputs(self, tags_and_types):
     # provided directly.
     assert self.side_input_maps is None
 
+    # Get experiments active in the worker to check for side input metrics exp.
+    experiments = RuntimeValueProvider(
```

Comment:
<!--new_thread; commit:ac6abcff9e381c9345f7aaeabdf8f83998f6074c; resolved:0-->
Can you make this a `set()` to match what you have in sideinputs.py?